repos/spack_repo/builtin/packages/llvm_amdgpu/package.py

```diff
@@ -66,6 +66,14 @@ class LlvmAmdgpu(CMakePackage, LlvmDetection, CompilerPackage):
 
     provides("c", "cxx")
     provides("fortran", when="@7.0:")
+    provides("fortran", when="+fortran")
+
+    variant(
+        "fortran",
+        default=True,
```

Comment:
Default should be false since spack-built llvm-amdgpu doesn't provide fortran.
```suggestion
        default=False,
```